OracleSOABPM/setenv.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+#
+#===============================================
+# MUST: Customize this to your local env
+#===============================================
+#
+# Directory where all domains/db data etc are 
+# kept. Directories will be created here
+export DC_USERHOME=/scratch/${USER}/docker
+
+# Registry names where requisite standard images
+# can be found
+export DC_REGISTRY_SOA="localhost"
+export DC_REGISTRY_DB="localhost"
+
+# Proxy Environment
+export http_proxy=""
+export https_proxy=""
+export no_proxy=""
+
+#===============================================
+exportComposeEnv() {
+  #
+  export DC_HOSTNAME=`hostname -f`
+  #
+  # Used by Docker Compose from the env
+  # Oracle DB Parameters
+  #
+  export DC_ORCL_PORT=1521
+  export DC_ORCL_OEM_PORT=5500
+  export DC_ORCL_SID=soadb
+  export DC_ORCL_PDB=soapdb
+  export DC_ORCL_SYSPWD=welcome1
```

Comment:
We don't allow hard-coded passwords anywhere. This needs to be changed to either accept a password or dynamically generate one.